install/kubernetes/microcks/values.yaml

```diff
@@ -52,6 +52,16 @@ microcks:
   # Enabling/disabling mock invocation stats.
   mockInvocationStats: true
 
+  # Enabling/disabling Custom keystore for JAVA
+  # Use an existing secret with the provided keystore
+  # You must concerve cacerts to avoid modifications on JAVA configuration
+  customKeystore:
```

Comment:
Can we change to simple:

```yaml
customSecretRef:
   secret:
   key:
```

We don't have to enable it, just put it in the values. By default, this block will be commented out in the `values.yaml` file.

Reply:
Thanks for the change @romain66 !